utils.py

```diff
@@ -13,6 +14,11 @@
 COLOR_UNDERLINE = '\033[4m'
 
 
+class Mode(Enum):
```

Comment:
I consider `utils.py` as a place for simple utility functions, but this seems more like a part of core logic. Better place it in `insomniac.py`